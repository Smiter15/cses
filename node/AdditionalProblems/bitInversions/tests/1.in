1111111111
10
1 7 6 6 5 1 5 8 5 8
