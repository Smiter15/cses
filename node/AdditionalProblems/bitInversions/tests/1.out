9 5 4 5 3 4 6 6 4 4 
